server/channels/web/static.go

```diff
@@ -78,6 +83,9 @@ func root(c *Context, w http.ResponseWriter, r *http.Request) {
 	staticDir, _ := fileutils.FindDir(model.ClientDir)
 	contents, err := os.ReadFile(filepath.Join(staticDir, "root.html"))
 	if err != nil {
+		c.Logger.Warn("Failed to read content from file",
+			mlog.String("file_path", filepath.Join(staticDir, "root.html")),
+			mlog.Err(err))
 		http.Error(w, err.Error(), http.StatusNotFound)
```

Comment:
For consistency, I think this should be an internal error rather than not found.